### PR TITLE
Adding Heroku Deploy button

### PR DESCRIPTION
**Pull request by BigAlUK**
Hi @elan,

I've added a Heroku Deploy button to try and make it easier for someone to get up and running with the app. 

Hope it helps.

Al

### CODE REVIEW

**Comment by elan** [comment]
Thanks, @BigAlUK, very kind of you 😀

File: README.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
-In order to run this app:
- 
-- Install [node.js](https://nodejs.org/en/).
-- Clone the repository.
-- Install dependencies using `npm install`.
-- Make a new app at Heroku, and add the Redis Cloud add-on (free plan) and note the app URL.
```
Comment (elan) [inline README.md]: Should we leave the first steps around for people wanting to develop locally?

Comment (BigAlUK) [inline README.md]: Good call. I was in two minds when I removed it, I'll put it back when I get to a machine.

**Comment by BigAlUK** [comment]
I've re-added the original README text (with a couple of updates). Hope that's ok.

**Comment by elan** [comment]
Looks perfect, thanks!

**Pull request merged**